packages/playground/compile.js

```diff
@@ -0,0 +1,6 @@
+import { readFileSync } from 'fs';
```

Comment:
it could be helpful to add something to the readme about how to use this file vs the other playground files for debugging or experimenting or whatever

Reply:
Done.